pegasus/sites.v3/hourofcode.com/public/thanks.md.partial

```diff
@@ -3,20 +3,8 @@ title: Thanks for signing up to host an Hour of Code!
 layout: wide
 nav: how_to_nav
 
-social:
-  "og:title": 'Hour of Code: Join the Movement'
-  "og:description": 'The Hour of Code is a global movement reaching tens of millions of students of all ages in 180+ countries and over 45 languages.'
-  "og:image": "http://{{ urls/request_host }}/images/hourofcode-2015-video-thumbnail.png"
-  "og:image:width": 1440
-  "og:image:height": 900
-  "og:url": "http://{{ urls/request_host }}"
-
-  "twitter:card": player
-  "twitter:site": "@codeorg"
-  "twitter:url": "http://{{ urls/request_host }}"
-  "twitter:title": 'Hour of Code: Join the Movement'
-  "twitter:description": 'The Hour of Code is a global movement reaching tens of millions of students of all ages in 180+ countries and over 45 languages.'
-  "twitter:image:src": "http://{{ urls/request_host }}/images/hourofcode-2015-video-thumbnail.png"
+"twitter:card": player
```

Comment:
Do these work without the `social:` header above?

Reply:
It works without the header.  I will add the header.  Thanks.

Reply:
I reproed the prior situation to understand why that might be the case, and it turns out when `social:` is missing, these actually don't work.  This line goes missing (because it's not supplied by `social_metadata.rb`) when `social:` isn't there:
```
<meta content="@codeorg" property="twitter:site">
```